packages/transaction-controller/package.json

```diff
@@ -37,9 +37,9 @@
     "@metamask/base-controller": "^3.2.3",
     "@metamask/controller-utils": "^5.0.2",
     "@metamask/eth-query": "^4.0.0",
-    "@metamask/gas-fee-controller": "^10.0.0",
+    "@metamask/gas-fee-controller": "^10.0.1",
```

Comment:
Should we add this to the changelog too?

Reply:
Fixed: https://github.com/MetaMask/core/commit/194c74e747285515e41738387b9b59d156b9b0fe